Agrega embi y prestamos por sectores

juansqw opened the pull request.

juansqw committed:
Agrega embi y prestamos por sectores

Johan-rosa commented on R/tasas_diarias.R:
```diff
@@ -0,0 +1,35 @@
+#' @examples
+#' get_fbkf()
+
+get_tasas_diarias <- function() {
```
Según veo, el año podría ser un argumento. Como está ahora solo descargaríamos las del 2023.

Johan-rosa commented on R/tasas_diarias.R:
```diff
@@ -0,0 +1,35 @@
+  )
+
+  data2 <- data |>
+    apply()
```
Veo que está incompleta la función. Creo que podemos quitarla de este archivo y agregarla en otro PR.

Johan-rosa replied:
https://github.com/Johan-rosa/databcrd/issues/31

Johan-rosa committed:
delete tasas diarias

Johan-rosa commented on R/embi.R:
```diff
@@ -0,0 +1,65 @@
+                                    "mensual", "trimestral", "anual"),
+                              y = c("day", "week", "month", "quarter", "year"))
+  
+  periodicidad <- per_table |> 
```
Esto luce más sencillo.

```r
  periodicidad <- switch(
    periodicidad,
    diario = "day",
    semanal = "week",
    mensual = "month",
    trimestral = "quarter",
    anual = "year",
    stop("Frecuencia no definida")
  )
```

Johan-rosa commented on R/embi.R:
```diff
@@ -0,0 +1,65 @@
+        file_path,
+        sheet = 1,
+        col_names = TRUE,
+        col_types = c("text", rep("numeric", 20)),
```
En la data habían unos valores extraños que hacían que diera error. Esos valores por lo visto fueron agregados por error por el administrador y no se dio cuenta.

Johan-rosa committed:
Adjust get_embi

It was failing due to flying values in the file. It doesn't use a fixed number of columns anymore

Johan-rosa committed:
Merge branch 'main' into indicadores-osd

Johan-rosa committed:
chore: add global variable

Johan-rosa committed:
Remove tidyselect namespace, use dplyr

Johan-rosa committed:
Add sectores as global

Johan-rosa committed:
Ignore words

Johan-rosa reviewed (commented):
LGTM
